compiler/resolution/visibleFunctions.cpp

```diff
@@ -62,6 +62,8 @@ static Map<BlockStmt*, BlockStmt*>            visibilityBlockCache;
 
```

Comment:
Is visibilityBlockCache still used?

If not, would you add a note to the PR message why we are better off without it.

Reply:
It didn't seem to have a significant impact on performance, and I think simpler is better unless we see measurable benefit.